rest_api/sawtooth_rest_api/route_handlers.py

```diff
@@ -499,6 +503,12 @@ def _check_status_errors(proto, content, error_traps=None):
         except AttributeError:
             pass
 
+        try:
+            if content.status == proto.INVALID_SORT:
+                raise errors.SortInvalid()
+        except AttributeError:
+            pass
```

Comment:
There is a comment on the first time I do this a few lines above. I could add a comment every time if you think it's important.

I'd also be open to a different approach. I have never been happy with this solution, and have gone over many alternatives in my head, but they all seem imperfect.